services/gps2tsdb/healthcheck.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/python3
+import postgres
+import sys
+import os
+import datetime
+
+threshold = 5
```

Comment:
This could be an environment variable.

Reply:
Def could, however unsure if this needs to be something that is user configurable. I guess if you have a gps unit that gets points less than every 5 seconds?